### PR TITLE
fix(upw): keyword for vertical K

### PR DESCRIPTION
**Pull request by ConstableCatnip**
The keyword for vertical K in mfupw.py was inconsistent with online documentation for upw and with mflpf.py. This was causing vertical Ks to be set to 0 when loading a model setup using parameters.

Tagging along in this PR is also a fix for the error message for the layer proportion check with multilayer hobs. Currently “.format” only operates on the last line of the error message instead of the entire string.

### CODE REVIEW

**Comment by langevin-usgs** [comment]
@ConstableCatnip, thanks for this PR, but do you think you could make the fix using the same approach that is in the LPF package load?  I'm sure your change is fine, but it might be nice to keep the approaches consistent.  Also, perhaps you could add a variation on the twrip example in examples/data/parameters that has a UPW package?

**Comment by ConstableCatnip** [comment]
Sure thing! Let me know if this is along the lines of what you were thinking.

**Comment by langevin-usgs** [comment]
Looks like a minor codacy issue.  You might also need to update from develop to get a recent fix for some python 3.8 issues.

File: flopy/modflow/mfupw.py [hunk 1]
```diff
@@ -276,7 +277,7 @@ def write_file(self, check=True, f=None):
             if self.chani[k] < 1:
                 f_upw.write(self.hani[k].get_file_entry())
             f_upw.write(self.vka[k].get_file_entry())
-            if transient == True:
+            if transient is True:
```
Comment (langevin-usgs) [inline flopy/modflow/mfupw.py]: I'm sorry to beat you up on this, but I think the correct syntax is:

`if transient:`

Comment (langevin-usgs) [reply flopy/modflow/mfupw.py]: And then I think it's ready to go.  Thanks so much.

Comment (ConstableCatnip) [reply flopy/modflow/mfupw.py]: No worries. codacy recommend either `if transient:` or `if transient is True:`. I had gone with the latter for readability.

**Pull request merged**